### PR TITLE
Add npm scripts for quick develop

### PR DESCRIPTION
**Pull request by lex111**
As an alternative to the two commands: `yarn start` + `yarn run android|ios`, instead just run `yarn start: android` or `yarn start:ios` - you do not need to run two terminals. Quickly and convenient.

Can I add the appropriate commands with `logger`?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "start:logger": "LOGGER_ENABLED=true node node_modules/react-native/local-cli/cli.js start --reset-cache",
+    "start:android": "concurrently -r 'node node_modules/react-native/local-cli/cli.js start' './node_modules/.bin/react-native run-android'",
```
Comment (alejandronanez) [inline package.json]: You can do `react-native run-android` instead of `./node_modules/.bin/react-native run-android`

Comment (andrewda) [inline package.json]: We can do that for a lot of things here if I'm not mistaken (e.g. `start` could just be `react-native start`)

Comment (alejandronanez) [inline package.json]: That's true. If you don't mind you can do it too here :)

Comment (lex111) [inline package.json]: Yes, I also thought about it. But I often saw that usually write  the commands with full paths (it seems, for greater compatibility)

Comment (alejandronanez) [inline package.json]: hmmm I haven't had any issue with that. Basically what node does is searching inside the `node_modules/.bin` folder for those executables (that should be installed after running `yarn`)

**Comment by alejandronanez** [comment]
About

> Can I add the appropriate commands with logger?

Can you clarify a little please?

File: package.json (hunk 1 above)
Comment (lex111) [inline package.json]: Perhaps the reason is that you will not have globally `react-native`?

Comment (alejandronanez) [inline package.json]: Your local modules will have precedence over the globally installed ones, so it's safe to remove the whole path.

Comment (alejandronanez) [inline package.json]: As long as you're using something that's inside `node_modules/.bin` you'll be ok.

**Comment by lex111** [comment]
> Can you clarify a little please?

@alejandronanez We still have a [command to run with a logger](https://github.com/gitpoint/git-point/pull/123/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R7), can add the same commands (with `concurrently`), but with the run of the logger?

**Comment by alejandronanez** [comment]
I think running the logger should be optional, if you want to add concurrently to it we should have the option for running it with `yarn start:ios:logger` for example.
It's a good use case, go for it. Can you please update the README.md file too with the new commands?

Thanks!

File: package.json [hunk 2]
```diff
@@ -3,11 +3,15 @@
...
-    "start:logger": "LOGGER_ENABLED=true node node_modules/react-native/local-cli/cli.js start --reset-cache",
-    "link": "./node_modules/.bin/react-native link",
-    "ios": "./node_modules/.bin/react-native run-ios",
-    "android": "./node_modules/.bin/react-native run-android",
+    "start": "react-native start",
+    "start:logger": "LOGGER_ENABLED=true react-native start --reset-cache",
+    "start:android": "concurrently -r 'react-native start' 'react-native run-android'",
+    "start:android:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start' 'react-native run-android'",
+    "start:ios": "concurrently -r 'react-native start' 'react-native run-ios'",
+    "start:ios:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start' 'react-native run-ios'",
```
Comment (andrewda) [inline package.json]: Do these need `react-native start --reset-cache`? That's what the `start:logger` script has, not sure if it's important.

Comment (lex111) [inline package.json]: Of course, fixed :+1:

**Comment by lex111** [comment]
@alejandronanez the scripts updated, with the update README.md could you help me? I'm afraid there will be a lot of duplication of the same words.

**Comment by lex111** [comment]
@housseindjirdeh great, I generally thought of leaving the old scripts, but it's probably better. And how to write a description for the logger - `yarn start:ios:logger`, do not duplicate the same?

**Comment by housseindjirdeh** [comment]
@lex111 we can probably just add it to the same line. For example:

- `yarn start:ios` to start the packager and run the app in the iOS simulator (`yarn start:ios:logger` will boot the application with [redux-logger](https://github.com/evgenyrodionov/redux-logger))
- `yarn start:android` to start the packager and run the app in the the Android device/emulator (`yarn start:android:logger` will boot the application with [redux-logger](https://github.com/evgenyrodionov/redux-logger))

Let me know if that works mate :raised_hands:

**Comment by lex111** [comment]
@housseindjirdeh these commands work, updated the description of the build process in all required files :heavy_check_mark:

**Pull request merged**